include/mega/posix/megafs.h

```diff
@@ -80,7 +80,7 @@ class MEGA_API PosixFileSystemAccess : public FileSystemAccess
     int defaultfilepermissions;
     int defaultfolderpermissions;
 
-    FileAccess* newfileaccess();
+    FileAccess* newfileaccess(bool followSymLinks = true);
```

Comment:
I will add some `override`s for the declaration of this virtual methods here, just to ensure we don't make a mistake changing the signature of the method at the parent class.